METADATA_SUPPORT.md

```diff
@@ -483,15 +483,19 @@ v55 introduces the following new types.  Here's their current level of support
 
 |Metadata Type|Support|Notes|
 |:---|:---|:---|
```

Comment:
Not sure how this magic happens.  commit flow I assume.  Lemme know if I need to fix up something here.